index.html

```diff
@@ -16,7 +16,7 @@
        <a href="#">Habeeb Musa Efiamotu</a>
       </div>
```

Comment:
### Your Logo

![navbar-logo (copy)](https://user-images.githubusercontent.com/93039359/178567547-56ff9060-57af-4ff0-91d7-d23b326aeba6.png)


- [ ] Please adjust the contrast of your logo in default mode. It should be 4.5:1 :blush: